criu/net.c

```diff
@@ -111,15 +111,18 @@ int read_ns_sys_file(char *path, char *buf, int len)
 	}
 
 	rlen = read(fd, buf, len);
+	if (rlen == -1)
+		pr_perror("Can't read ns' %s", path);
 	close(fd);
```

Comment:
I would move it to after the `close` and add `return -1`. Being explicit always helps.